session-artifacts/handoff-manual-actions.html

```
@@ -0,0 +1,380 @@
+<title>Issue #418 — handoff</title>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)